### PR TITLE
Institutional landing page: adding AEM instructions

### PR DESCRIPTION
**Pull request by Garneauma**
Adding instructions for AEM users on how to implement the page template.

Dependent on https://github.com/wet-boew/wet-boew/pull/9758 being merged

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
See the inline suggestion. 

I reviewed and other than the suggestion, everything looks good from a technical perspective. I confirm this is a content change.

As discussed today, we will wait for PP to review the wording prior to merge this PR.

**Comment by duboisp** [comment]
I will follow up internally about the wording to unblock this PR.

File: templates/institutional-landing/index.json-ld
Comment (duboisp) [inline templates/institutional-landing/index.json-ld:176]:
```suggestion
					"Create a page with the <strong>Generic template</strong>",
					"Applying Content type (navigation page – institutional profile) via the page <strong>Properties</strong> -> <strong>Basic tab</strong> -> <strong>Content type</strong>, set the value to “<strong>navigation page – institutional profile</strong>”.",
					"Configure Page container manually via the page <strong>Properties</strong> -> <strong>Template options</strong> tab -> <strong>Advanced</strong>, check “<strong>Page container: Configure manually</strong>”.",
					"Configure your ILP page. The ILP page contains several GCWeb components. Each component has its own configuration which you can adapt as per your institutional needs. Please see individual component for more details about their configurable options."
					"Remove the <strong>H1 Title</strong>.",
					"Add the ILP HTML code with a <strong>Generic HTML component</strong> component.",
					"Edit the ILP HTML code"
				],
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — CHANGES_REQUESTED** [review]
The instruction looks good and looks to be aligned with the formal AEM instruction.

See the additional change in the inline comments.

The item regarding the data-ajax are out of scope from this PR but we should submit a fix for the next wet-boew release.

In this PR, can you remove the link toward the working copy of the institution landing page which was linked from its old location. I am referring to the following line of code:
https://github.com/wet-boew/GCWeb/blob/master/templates/institutional/index.json-ld#L52-L61

File: templates/institutional-landing/index.json-ld
Comment (Garneauma) [reply templates/institutional-landing/index.json-ld:193]: This solution seems to work well.

File: templates/institutional-landing/institutional-landing-bad.html
Comment (duboisp) [inline templates/institutional-landing/institutional-landing-bad.html:9]: This dont' work. The custom JS are not triggered to fix the heading by adding the page body CSS class `page-type-ilp`

**Review by duboisp — CHANGES_REQUESTED** [review]

File: package.json
Comment (duboisp) [reply package.json:16]: You re-added this change by mistake, please remove this file change. Thanks

**Review by duboisp — CHANGES_REQUESTED** [review]
Tested and reviewed, everything look good except the change in the package.json need to be reverted.

**Review by duboisp — APPROVED** [review]
Appoved

**Pull request merged**
